server/views/partials/layout.njk

```diff
@@ -13,6 +13,11 @@
   <script src="/assets/js/html5shiv-3.7.3.min.js"></script>
   <![endif]-->
 
+  <script type="application/javascript">
+      window.gaConfig = {};
+      window.gaConfig.uaId = "{{ uaId }}";
```

Comment:
As each environment will require its own UA- id, if possible, can this value be added at build time via a ENV in the CI build?

Reply:
yep that is correct and would work, exactly how we do it in our service here. Will take a go at adding that now, thanks